homeassistant/components/zwave_js/light.py

```diff
@@ -279,6 +282,11 @@ async def async_turn_on(self, **kwargs: Any) -> None:
                 rgbw_channels[ColorComponent.COLD_WHITE] = rgbw[3]
             await self._async_set_colors(rgbw_channels, transition)
 
+        if kwargs.get(ATTR_BRIGHTNESS) is None:
```

Comment:
Why do we have this check?

Reply:
When you only toggle the light on no brightness is sent, and we send zwavejs a 255 to "toggle on to last brightness".

Reply:
I guess I could move it into `_async_set_brightness` since we're already checking in there.

Reply:
In the case when brightness is set, the driver will do its own optimistic reply with "yeah, the level is at N", and so the state is updated almost immediately already. It didn't seem necessary to do two state updates.

Reply:
I moved the calls under `_async_set_brightness` since there's existing checks there for the brightness being `None`.